desktop/core/src/desktop/static/desktop/js/select2.full.patched.js

```diff
@@ -16,6 +16,10 @@
 
 // see https://github.com/select2/select2/issues/3992 for the patch, hopefully it's gonna be fixed in a next release of Select2
 
+// Note on Cross-site scripting (XSS)
+// This legacy version of select2 is only used via the Knockout.js binding in ko.select2.js
+// where a patch for XSS is applied. 
+
```

Comment:
Good to note the CVE identifier here as well

Reply:
Good point, there was no official CVE attached to the jira, but I could find it at 
https://nvd.nist.gov/vuln/detail/CVE-2016-10744. Will update the code comment.